__tests__/app.test.js

```diff
@@ -48,13 +41,50 @@ describe("api/topics", () => {
   });
 });
 
+describe("api/articles", () => {
+  test("GET:200 returns an array of article objects with corresponding properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(articles.length).toBe(13);
+        expect(articles[0].comment_count).toBe("2");
+        articles.forEach((article) => {
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
+          expect(article.hasOwnProperty("body")).toBe(false);
+        });
+      });
+  });
+  test("GET:404 returns status 404 for a non-existent path", () => {
+    return request(app).get("/api/article").expect(404);
```

Comment:
would prefer you to also expect a message for this error too. Later on we will be making more 404 errors and we want to know what kind of 404 these are so the message will help us differentiate it